sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryClient.java

```diff
@@ -8,27 +8,28 @@
 import com.azure.core.http.rest.SimpleResponse;
 import com.azure.core.http.rest.VoidResponse;
 import com.azure.core.util.Context;
+import com.azure.storage.common.Utility;
 import com.azure.storage.common.credentials.SASTokenCredential;
 import com.azure.storage.common.credentials.SharedKeyCredential;
 import com.azure.storage.file.models.DirectoryInfo;
 import com.azure.storage.file.models.DirectoryProperties;
 import com.azure.storage.file.models.DirectorySetMetadataInfo;
 import com.azure.storage.file.models.FileHTTPHeaders;
+import com.azure.storage.file.models.FileInfo;
 import com.azure.storage.file.models.FileRef;
 import com.azure.storage.file.models.HandleItem;
 import com.azure.storage.file.models.StorageException;
 import java.net.URL;
+import java.time.Duration;
 import java.util.Map;
+import reactor.core.publisher.Mono;
 
 /**
  * This class provides a client that contains all the operations for interacting with directory in Azure Storage File Service.
  * Operations allowed by the client are creating, deleting and listing subdirectory and file, retrieving properties, , setting metadata
  * and list or force close handles of the directory or file.
- *
```

Comment:
Were all these new line removals triggered by IntelliJ's auto formatting or done manually? Removing them makes the comments harder to read when viewing them in an IDE (they will generate the same Javadocs luckily)